.github/workflows/publish.yaml

```diff
@@ -0,0 +1,107 @@
+# the publish workflow builds zos binaries
+# and tag them correctly
+# the build is triggered when pushing to any
+# branch, but the tagging of the build only happens
+# either on main or when a tag is created
+name: Release
+on:
+  push:
+    branches:
+      - "*"
```

Comment:
I am curious on why we are publishing on every push on any branch. I understand for tags and maybe on main, but why all branches?

Reply:
This indeed will trigger the build for all branches, but later in the workflow we have checks for specific patterns to make sure tagging of flists only happens on main and a tagged branch. 

This is to make sure all builds are working fine on that branch. We can later improve on this and only build on "development" branches if the workflows or the binaries has modified.